src/styles/global.css

```diff
@@ -0,0 +1,22 @@
+@import 'colors.css';
```

Comment:
`@import` prevents stylesheets from being loaded concurrently, so just add both stylesheets in each html file